nemo/nemo/backends/pytorch/actions.py

```diff
@@ -14,6 +14,7 @@
 import torch.nn as nn
 import torch.optim as optim
 from nemo.backends.pytorch.nm import TrainableNM
+from apex.optimizers import FusedLAMB
```

Comment:
This will fail when user does not have GPUs (e.g. trying things out on their MacBook) or inference-only on CPU.
Please follow the same pattern as we do below for DDP.
You can throw an exception is there is no CPU-based LAMB

Reply:
ac988aa6cb04d24088f168778dafb21e5612e1c7